app/src/core/design/components/userComponents/UtilComponents/Border.js

```diff
@@ -11,24 +11,37 @@ const useStyles = makeStyles(theme => ({
     }
 }));
 
-function Border({ propKey, propName, setProp, props, styleProp }) {
+function Border({ propKey, propName, setProp, props, styleProp, type }) {
     const classes = useStyles();
     const borderStylesOpts = ["Solid", "Dotted", "Dashed"];
     const tmp = props[styleProp][propKey];
     let [width, style, color] = typeof tmp === "string" ? [...tmp.split(" ")] : [];
     width = width && width !== "" ? width : "0px";
     style = style && style !== "" ? style : "solid";
     color = color && color !== "" ? color : "#000000";
+
     const selectedValue = borderStylesOpts.filter(opts => opts.toLowerCase() === style);
+
     const handleSetProp = () => {
         setProp(props => {
             if (propName === "All Sides") {
-                props[styleProp]["borderTop"] = width + " " + style + " " + color + " ";
-                props[styleProp]["borderBottom"] = width + " " + style + " " + color + " ";
-                props[styleProp]["borderRight"] = width + " " + style + " " + color + " ";
-                props[styleProp]["borderLeft"] = width + " " + style + " " + color + " ";
+                if (type === "outlined" && width === "0px") {
```

Comment:
we have do this in the button file. changing border.js is not required. 
check the same condition in Button.js and ButtonExport